multiview/predict/base.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2019 NeuroData (http://neurodata.io)
+#
```

Comment:
I didn't have a good place of putting this comment so I'm putting it here. I don't like "predict" as a module name since there's lots of things that deal with prediction. How sklearn does it is to just call out the overall algorithm idea. So for this case, I would change it to cotraining (i.e. "multiview/cotraining/CTClassifier"

Reply:
i changed the module name as you suggested